h/static/scripts/filter/test/document-domain-test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+describe('documentDomain', function() {
+
+  before(function() {
+    angular.module('h', []).filter(
```

Comment:
Given the simplicity of these modules, it might be better to test them without any reference to angular at all...